pages/en/billing.mdx

```diff
@@ -63,3 +63,60 @@ title: Billing on Subgraph Studio
 5. Click 'Withdraw GRT' to withdraw the GRT from your account. Sign the associated transaction in your wallet. This will cost gas.
 
 6. Once the transaction is confirmed, you'll see the GRT added to your Arbitrum wallet within an hour.
+
+## Getting GRT
+
+This section will show you how to get GRT to pay for query fees.
+
+### Coinbase
+
+This will be a step by step guide for purchasing GRT on Coinbase.
+
+1. Go to [Coinbase](https://www.coinbase.com/) and create an account.
+2. Once you have created an account, you will need to verify your identity. This is a standard procedure for all crypto exchanges.
```

Comment:
This is a standard procedure for all centralized or custodial crypto exchanges.

Reply:
Nit picky comment, but decentralized exchanges don't require KYC.

Reply:
Not nit picky at all - that's a good catch! Thank you!